qiskit/aqua/operators/primitive_ops/pauli_op.py

```diff
@@ -306,6 +306,10 @@ def to_instruction(self) -> Instruction:
 
         return self.to_circuit().to_instruction()
 
+    # def to_gate(self) -> Gate:
```

Comment:
Why is this commented?

Reply:
I rewrote this method so it's not commented out anymore!